packages/story-editor/src/components/canvas/karma/selection.karma.js

```diff
@@ -89,7 +89,7 @@ describe('CUJ: Creator can Transform an Element: Selection integration', () => {
       up(),
     ]);
     const bgFrame = fixture.editor.canvas.framesLayer.frames[0].node;
-    expect(await getSelection()).toEqual([bgFrame.dataset.elementId]);
+    expect(await getSelection()).not.toEqual([bgFrame.dataset.elementId]);
```

Comment:
hmph, possibly better to see that selection remains on the `frame1` element than not on the bg element? just thinking it might make the expected behavior a little more explicit? probs not a biggie either way
```js
expect(await getSelection()).toEqual([frame1.dataset.elementId]);
```